docs/topics/multiplatform/multiplatform-dsl-reference.md

```diff
@@ -36,24 +36,17 @@ plugins {
-| `targets`            | All targets of the project.                                                                                                                                                                                                     |
-| `presets`            | All predefined targets. Use this for [configuring multiple predefined targets](#targets) at once.                                                                                                                               |
-| `sourceSets`         | Configures predefined and declares custom [source sets](#source-sets) of the project.                                                                                                                                           |
-| `compilerOptions`    | Extension-level common [compiler options](gradle-compiler-options.md) that are used as defaults for all targets and shared source sets. To use it, add the following opt-in: `@OptIn(ExperimentalKotlinGradlePluginApi::class)` |
-
-> The support for `compilerOptions {}` as a top-level block is [Experimental](components-stability.md#stability-levels-explained)
-> and requires opt-in. It may be dropped or changed at any time. Use it only for evaluation purposes. We would appreciate
-> your feedback on it in [YouTrack](https://kotl.in/issue).
->
-{style="warning"}
+| **Block**            | **Description**                                                                                                                |
+|----------------------|--------------------------------------------------------------------------------------------------------------------------------|
+| _&lt;targetName&gt;_ | Declares a particular target of a project. The names of available targets are listed in the [Targets](#targets) section.       |
+| `targets`            | All targets of the project.                                                                                                    |
+| `sourceSets`         | Configures predefined and declares custom [source sets](#source-sets) of the project.                                          |
+| `compilerOptions`    | Extension-level common [compiler options](#compiler-options) that are used as defaults for all targets and shared source sets. |
 
 ## Targets
 
-_Target_ is a part of the build responsible for compiling, testing, and packaging a piece of software aimed at
-one of the supported platforms. Kotlin provides target presets for each platform. See how to [use a target preset](multiplatform-set-up-targets.md).
+A _target_ is a part of the build responsible for compiling, testing, and packaging a piece of software aimed at
+one of the supported platforms. Kotlin provides targets for each platform. See [Set up targets for Kotlin Multiplatform](multiplatform-set-up-targets.md).
```

Comment:
Is it just me, or is there a little bit of a non sequitur? "Kotlin provides targets for each platform. See how to set up targets." But I thought Kotlin provides them?..

Do you think we can include a transition here that would explain the use case, a reason to follow this link?

Reply:
What do you think about:
A _target_ is a part of the build responsible for compiling, testing, and packaging a piece of software for
one of the supported platforms. Kotlin provides targets for each platform so you can instruct Kotlin to compile code for that specific target. To learn more about how to  configure these targets, see [Set up targets for Kotlin Multiplatform](multiplatform-set-up-targets.md).

Reply:
Yep, this way it's immediately clear that it's about configuration and not setting up new ones.